.circleci/config.yml

```diff
@@ -624,6 +627,10 @@ jobs:
           keys:
             - v3-lambdas-convert-dependencies-{{ checksum "yarn.lock" }}
             - v3-lambdas-convert-dependencies-
+      - run:
+          name: Install poppler (needed for pdf conversion)
+          command: make env.d/lambda && bin/install_poppler_circle-ci.sh
```

Comment:
After some considerations I think we should use the generated docker image previously build in `build-lambda-docker` job because this will ensure that the image itself is working with this specific version of poppler. It will be also less error prone, we will manipulate only once the installation and not twice on two different linux OS.